### ISSUE TITLE
avis home page-Check Boxes in Accessibility menu are not labeled

### ISSUE THREAD

**Issue by nsingh2001** [Kishan1601/Test-1#8]
UAT 2022-03
Windows 10
Chrome Version 98.0.4758.102 (Official Build) (64-bit)
NVDA 2021.3.5
Repro Steps
1.	Open the URL https://www.avis.com/en/home

2 Press Ctrl + f11 to enable the Accessibility menu, Press Tab to navigate to the Accessibility menu option 
2.	press enter to open Accessibility option, 
3.	4.press tab to find all the 6 check boxes

Expected Result check box should be announce by the screen reader
Actual Result
All check box are not labelled properly and not announced by screen reader
WCAG Guideline
https://www.w3.org/TR/WCAG21/#name-role-value
